### ISSUE TITLE
[web] New Role "Secure View"

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#10765]
# Description

## User Stories

* > As a CISO I want to make sure that documents can be shared without the risk of a user caused data loss.

## Acceptance Criteria
- [x] prerequisite: share role from the server - https://github.com/owncloud/ocis/pull/8907
- [x] prerequisite: missing download permission from the server (WebDav permission - will be the indicator, if a resource is shared via secure view. if the download permission is not set, this will indicate secure view) - https://github.com/cs3org/reva/pull/4694
- [x] (add icon for new secure view role "shield-fill" )
- [x] Download permission and action is not available
- [x] copy action is not available (not in the context menu, not via shortcut)
- [x] files can only be opend with collabora; there is no other option to open files eg. with the pdf preview or md editor etc.
- [x] documents are being opened in secure view in collabora - https://github.com/owncloud/ocis/issues/9068
- [x] no previews are loaded when download permission is missing
- [ ] add e2e tests

## Definition of ready
- [ ] Everybody needs to understand the value written in the user story
- [ ] Acceptance criteria have to be defined
- [ ] All dependencies of the user story need to be identified
- [ ] Feature should be seen from an end user perspective
- [ ] Story has to be estimated
- [ ] Story points need to be less than 20

## Definition of done
- Functional requirements
  - [ ] Functionality described in the user story works
  - [ ] Acceptance criteria are fulfilled
- Quality
  - [ ] Code review happened
  - [ ] CI is green (that includes new and existing automated tests)
  - [ ] Critical code received unit tests by the developer
- Non-functional requirements
  - [ ] No sonar cloud issues

## Visual
![screenshot_002554](https://github.com/owncloud/web/assets/26610733/2da80720-3030-4623-ad45-c60807b08eb2)

**Comment by AlexAndBear** [comment]
I cant continue with this sprint item, because it seems like the server still sets the download permission. 

Need this fixed to get unblocked 

@butonic @butonic

**Comment by AlexAndBear** [comment]
transfered e2e test part to new ticket https://github.com/owncloud/web/issues/10985
